packages/edit-site/src/components/sidebar/global-styles-v2/components/screen.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * WordPress dependencies
+ */
+import { Surface } from '@wordpress/components';
+
+export const Screen = styled( Surface, { props: { variant: 'tertiary' } } )`
```

Comment:
What's the `variant: tertiary` expressing?

Reply:
It appears to be a mistake: I can't get this API to do anything: https://codesandbox.io/s/competent-grothendieck-weds7?file=/src/App.js

Nor is it documented in Emotion's documentation or types.

Reply:
I believe it's a prop of `Surface`

https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/surface/styles.js#L51

Reply:
From the README: 

```
tertiary: Used as the app/site wide background. Visible in dark mode only. Use case is rare.
```

Reply:
Yes it's a prop of Surface but the actual usage here doesn't make any sense or do anything it seems.